h2o-algos/src/main/java/hex/tree/SharedTree.java

```diff
@@ -191,6 +191,9 @@ protected void ignoreInvalidColumns(int npredictors, boolean expensive) {
       warn("_parallel_main_model_building",
               "Parallel main model will be disabled because use_best_cv_iteration is specified.");
     }
+    if (_parms._build_tree_one_node) {
+      warn("_build_tree_one_node", "Single-node tree building is not supported in this version of H2O.");
```

Comment:
So what is the reason to keep this parameter? It was supported in the past and now it is deprecated or it was not never implemented?

Reply:
It used to work but stopped working at some point. The parameter is not deprecated - just doesn't do anything currently. Instead of deprecating we should fix it - https://h2oai.atlassian.net/browse/PUBDEV-8564

Reply:
So if I understand correctly, this PR is just a quick fix and another one with a real fix will be implemented later? 

Maybe some description of PR with the JIRA link would help. ;)

Reply:
I thought the jira already has the information - https://h2oai.atlassian.net/browse/PUBDEV-8563 - what do you think I should add?

Reply:
Sorry, I missed it, my bad.